lib/stoplight/notifier/slack.rb

```diff
@@ -0,0 +1,29 @@
+# coding: utf-8
+
+module Stoplight
+  module Notifier
+    # @see Base
+    class Slack < Base
+      # @return [Proc]
+      attr_reader :formatter
+
+      # @return [::Slack::Notifier]
+      attr_reader :slack_notifier
+
+      # @param slack_notifier [::Slack::Notifier]
+      # @param formatter [Proc, nil]
+      def initialize(slack_notifier, formatter = nil)
+        @slack_notifier = slack_notifier
+        @formatter = formatter || Default::FORMATTER
+      end
+
+      def notify(light, from_color, to_color, error)
+        message = formatter.call(light, from_color, to_color, error)
+
+        slack_notifier.ping(message)
+
+        message
+      end
```

Comment:
I would prefer removing the blank lines here, like in [`Stoplight::Notifier::HipChat#notify`](https://github.com/orgsync/stoplight/blob/v1.0.0/lib/stoplight/notifier/hip_chat.rb#L36-L40).